packages/js/create-product-editor-block/plugin-templates/.wp-env.override.json.mustache

```diff
@@ -0,0 +1,18 @@
+{
+	"phpVersion": "8.2",
+	"core": "https://wordpress.org/wordpress-6.2.1.zip",
+	"plugins": [
+		"https://github.com/woocommerce/woocommerce/releases/download/nightly/woocommerce-trunk-nightly.zip",
+		"https://github.com/woocommerce/woocommerce/releases/download/wc-beta-tester-2.3.0/woocommerce-beta-tester.zip",
```

Comment:
Related to the below comment, this means the beta tester is not required anymore.

Reply:
While not required, it seems like it could still be a good idea to include, since it exposes 3PDs to the beta tester, which can be a helpful tool. What do you think?

Reply:
Yeah I think that makes sense, sounds good with me :)